sky/data/storage.py

```diff
@@ -893,8 +892,8 @@ def _get_bucket(self) -> Tuple[StorageHandle, bool]:
                 command = f'aws s3 ls {self.name}'
                 with ux_utils.print_exception_no_traceback():
                     raise exceptions.StorageBucketGetError(
-                        _BUCKET_FAIL_TO_CONNECT_MESSAGE.format(
-                            name=self.name, command=command)) from e
+                        _BUCKET_FAIL_TO_CONNECT_MESSAGE.format(name=self.name) +
+                        f' Consider using {command} to debug.') from e
```

Comment:
nit: How about `To debug, consider using {command}`, to make it easier to copy the command?

Reply:
Done, thanks!